salt/etcd/init.sls

```diff
@@ -39,9 +59,9 @@ etcd:
     - proto: tcp
   caasp_service.running_stable:
     - name: etcd
-    - successful_retries_in_a_row: 50
-    - max_retries: 300
-    - delay_between_retries: 0.1
+    - successful_retries_in_a_row: 10
+    - max_retries: 30
+    - delay_between_retries: 1
```

Comment:
Why we changed this? In a healthy cluster this will take 10 seconds at least per etcd instance, whereas before it was 5 seconds. Also, before we had a wider window to check if `etcd` was running in a stable manner.

Reply:
In my expericne, 5 seconds is not enough time for knowing if the server is up and running (for example, it can timeout not reaching consensus).

And retrying these checks after 100ms does not provide anything (besides overloading the machine). It is better to just check less but for more time.

And adding 5 seconds per etcd instance is not so bad. They are all provisioned in parallel, so we are adding 5 seconds to the whole orchestration...

Reply:
100ms is not overloading the machine at all (in fact it's still asking for more work), but OK.